bitbots_ros_control/src/imu_hardware_interface.cpp

```diff
@@ -21,7 +21,7 @@ ImuHardwareInterface::ImuHardwareInterface(rclcpp::Node::SharedPtr nh,
   name_ = name;
   diag_counter_ = 0;
   imu_msg_ = sensor_msgs::msg::Imu();
-  imu_msg_.header.frame_id = "imu_frame";
+  imu_msg_.header.frame_id = frame_;
```

Comment:
Why is this `frame_` and not `frame`?

Reply:
You could do both. One is the class variable and one is the parameter.